token/cli/src/clap_app.rs

```diff
@@ -763,11 +763,34 @@ pub fn app<'a, 'b>(
                         .value_names(&["FEE_IN_BASIS_POINTS", "MAXIMUM_FEE"])
                         .takes_value(true)
                         .number_of_values(2)
+                        .hidden(true)
                         .help(
                             "Add a transfer fee to the mint. \
                             The mint authority can set the fee and withdraw collected fees.",
                         ),
                 )
+                .arg(
+                    Arg::with_name("transfer_fee_basis_points")
+                        .long("transfer-fee-basis-points")
+                        .value_names(&["FEE_IN_BASIS_POINTS"])
```

Comment:
We can also add a bit more here so that clap catches more issues for us
```suggestion
                        .value_names(&["FEE_IN_BASIS_POINTS"])
                        .conflicts_with("transfer_fee")
                        .requires("transfer_fee_maximum_fee")
```